src/ec/packages/ec-component-datepicker/datepicker.story.js

```diff
@@ -1,35 +1,66 @@
 /* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
-import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
 import withCode from '@ecl-twig/storybook-addon-code';
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
```

Comment:
You might not have noticed this, but we have from the story-utils package also a function returning the form knobs which is what you might want to use here instead of redefining all the knobs in this story. 
`getFormKnobs()`

Furthermore, you set these boolean as "use cases" while in all the other form elements these are in the group "states", you will fix this by just using the right function from the utilities, by the way.